azure-pipelines.yml

```diff
@@ -130,18 +130,35 @@ jobs:
     displayName: Style-Lint FPV Verilog source files with Verible
 
 - job: sw_build
-  displayName: Build Software for Earl Grey toplevel design
+  displayName: Build and test Software for Earl Grey toplevel design
```

Comment:
super minor nit: I always have a hard time figuring out which job is which on the azure pipelines page because the names are long and they don't fit the column. Can we consider shortening them (in this PR or later)? E.g., "SW build & test Earl Grey" or even "Earl Grey Software".

Reply:
Yes! Thanks for bringing that up, it's been bothering me too

Reply:
Ok, I've taken a pass at this, once I'm convinced they all rrender I'll re-request review for the new names